tests/Unit/NumericalAlgorithms/Spectral/Test_LegendreGaussLobatto.cpp

```diff
@@ -263,21 +264,21 @@ SPECTRE_TEST_CASE(
     "Unit.Numerical.Spectral.LegendreGaussLobatto.SpectralToGridPoints",
     "[NumericalAlgorithms][Spectral][Unit]") {
```

Comment:
Same in this file actually

Reply:
And I had missed some classes in `Spectral.cpp`. A `git grep -i "grid.*points.*to.*spectral"` comes up with nothing now :)